class/defaults.yml

```
@@ -96,6 +96,12 @@ parameters:
       customMetrics:
         enabled: false
         extraLabels: {}
+      ignoreDownscalingSync: false
+      schedule:
+        enabled: false
+        enableExpression: '0 22 * * 2'
+        disableExpression: '0 1 * * 3'
```

Maybe `schedule` is the better naming than `expression`? No hard feelings either way

```suggestion
        enableSchedule: '0 22 * * 2'
        disableSchedule: '0 1 * * 3'
```

---

I was torn between the two options, but opted for expression to be less redundant, since the parent object is already named schedule.